components/experimental/src/duration/mod.rs

```diff
@@ -0,0 +1,11 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+//! Duration formatting
+
+#![warn(missing_docs)]
+
+pub mod duration;
+
+pub use duration::Duration;
```

Comment:
I do not this that we use a re-export statement in ICU4X.

@sffc do we use a re-export statement in ICU4X ?

Reply:
Public symbols should generally be exported one time; otherwise, docs and FFI get messed up. It's okay to have `pub use` but only if pulling from a private module.

I'd suggest `mod duration;` and `pub use duration::Duration;`